dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -255,7 +252,7 @@ module istioIngressGatewayIPAddress '../modules/network/publicipaddress.bicep' =
     name: istioIngressGatewayIPAddressName
     ipTags: istioIngressGatewayIPAddressIPTagsArray
     location: location
-    zones: zoneRedundancyZones
+    zones: locationHasAvailabilityZones ? locationAvailabilityZones : null
```

Comment:
nit: could set this as a variable at the top, right?

Reply:
I think it reads better without another variable which may not convey the two possible outcomes